cmake/configs/base.cmake

```diff
@@ -121,7 +121,7 @@ config_choice(
         "pthread;SPINLOCK_PTHREAD_MUTEX;HAVE_LIBPTHREAD"
         "gcc;SPINLOCK_GCC;"
         "msvc;SPINLOCK_MSVC;WT_WIN"
-        "pthread_adaptive;SPINLOCK_PTHREAD_ADAPTIVE;HAVE_LIBPTHREAD"
+        "pthread_adaptive;SPINLOCK_PTHREAD_MUTEX_ADAPTIVE;HAVE_LIBPTHREAD"
```

Comment:
In [mutex.h](https://github.com/wiredtiger/wiredtiger/blob/develop/src/include/mutex.h#L88-L97), `SPINLOCK_PTHREAD_ADAPTIVE` is defined as well, should we keep both? What about the two other values `SPINLOCK_GCC` and `SPINLOCK_MSVC`?

Reply:
Those definitions are in the above three lines.

Reply:
I am... blind, sorry! 👀